python/cuml/ensemble/randomforest_shared.pxd

```diff
@@ -99,8 +97,7 @@ cdef extern from "cuml/ensemble/randomforest.hpp" namespace "ML":
     cdef void build_treelite_forest[T, L](ModelHandle*,
                                           RandomForestMetaData[T, L]*,
                                           int,
-                                          int,
-                                          vector[unsigned char] &) except +
+                                          int) except +
 
     cdef vector[unsigned char] save_model_protobuf(ModelHandle) except +
```

Comment:
we dont use this function anymore, it would be nice to not have it in the .pxd file

Reply:
Will remove.